readme.md

```diff
@@ -3,7 +3,7 @@
 ## Learning Objectives
 
 - Explain the importance of OOJS
-- Describe the role of ES2015 Classes and how they work
+- Describe the role of ES6 Classes and how they work
```

Comment:
why did you change to ES6? ES2015 is the accurate, official name. https://bytearcher.com/articles/es6-vs-es2015-name/